<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>506</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Visualisation</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Veuillez sélectionner les maladies:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_cb"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_features">
         <item row="3" column="1">
          <widget class="QComboBox" name="Disease_2"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="Param">
           <property name="text">
            <string>Maladie 3</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="Feature">
           <property name="text">
            <string>Maladie 2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_param_1">
           <property name="text">
            <string>Veuillez faire un choix</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="Disease_1"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="Stat">
           <property name="text">
            <string>Maladie 4</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="Experiment">
           <property name="text">
            <string>Maladie 1</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1" colspan="2">
          <widget class="QPushButton" name="maj_button">
           <property name="text">
            <string>Mettre à jour le Graphe</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="Disease_3"/>
         </item>
         <item row="6" column="1">
          <widget class="QTextBrowser" name="DocBrowser_1"/>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="Disease_4"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="AKERNEL_Button">
         <property name="text">
          <string>Analyse à noyaux</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ASPARSE_Button">
         <property name="text">
          <string>Analyse sparse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ACP_Button">
         <property name="text">
          <string>Analyse en composantes principales</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_machine_learning"/>
       </item>
      </layout>
     </item>
     <item row="2" column="3">
      <layout class="QVBoxLayout" name="verticalLayout_plot">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
